instrumentation/elastic-opentelemetry-instrumentation-openai/README.md

```diff
@@ -22,32 +22,31 @@ pip install elastic-opentelemetry-instrumentation-openai
 
 This instrumentation supports *zero-code* / *autoinstrumentation*:
 
-```
-opentelemetry-instrument python use_openai.py
+You can see telemetry from this package if you have an OpenTelemetry collector started, for example
+as documented in the root [examples](../../examples/) folder.
```

Comment:
This does not exist

Reply:
thanks for the help above. I'll remove this part